static/app/views/automations/components/automationActionSummary.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import {IconCircledNumber} from 'sentry/components/iconCircledNumber';
+import {ActionMetadata} from 'sentry/components/workflowEngine/ui/actionMetadata';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {ActionType} from 'sentry/types/workflowEngine/actions';
+
+type AutomationActionSummaryProps = {
+  actions: ActionType[];
+  className?: string;
+  hasTooltip?: boolean;
+};
+
+export function AutomationActionSummary({
+  className,
+  actions,
+  hasTooltip,
+}: AutomationActionSummaryProps) {
+  if (actions.length === 0) {
+    return t('No actions');
+  }
+
+  if (actions.length === 1 && actions[0]) {
+    const action = actions[0];
+    const metadata = ActionMetadata[action];
+    if (!metadata) {
+      return t('1 action');
+    }
+    const {name, icon} = metadata;
+    return (
+      <Flex align="center" gap={space(0.75)} className={className}>
+        <IconContainer>{icon}</IconContainer>
+        {name}
+      </Flex>
+    );
+  }
+
+  const actionsList = actions
+    .map(action => ActionMetadata[action]?.name)
+    .filter(x => x)
+    .join(', ');
+
+  return (
+    <ActionContainer align="center" gap={space(0.75)} className={className}>
+      <IconContainer>
+        <IconCircledNumber number={actions.length} />
+      </IconContainer>
+      <Tooltip title={actionsList} disabled={!hasTooltip} showUnderline={hasTooltip}>
```

Comment:
does this mean there will be an actions tooltip within the automation tooltip if there are multiple actions? 😅

Reply:
I added the `hasTooltip` prop so that we could disable it specifically! Because a tooltip in a tooltip would be terrible hah